### ISSUE TITLE
order fufilliment

### ISSUE THREAD

**Issue by tongalalal** [allegro/allegro-api#9669]
I need to modify the status of the order through the API,
1. Modify the order status from now to READY_FOR_SHIPMENT, what conditions are needed during this period, or can you directly https://api through the interface. {environment}/order/checkout-forms/{id}/fulfillment,
2. Here I https://api through this interface. {"errors":[{"code":"NotSupportedException","message":"Unsupported content type. Please check 'Content-Type' request header","details":null,"path":null,"userMessage":"Request contains invalid data. Contact the application author.","metadata":{}}]}, why is that? How do I change this? Here's my trace_id: a9ae006d51028e51
Please answer these two questions of mine, thank you very much

**Comment by MaciejFrackowiak** [comment]
1. It all depends on the seller's order management - you can switch fulfillment status anytime.
2. You didn't provide a correct Content-type header - the right one is "application/vnd.allegro.public.v1+json".

**Comment by MaciejFrackowiak** [comment]
No, the package should be generated by the seller - we have [Shipment management](https://developer.allegro.pl/documentation/#tag/Shipment-management) resources to prepare the package, label, etc.

**Comment by tongalalal** [comment]
@MaciejFrackowiak That is to say, no matter which logistics cooperation model is chosen, does the seller need to create the information of the package themselves

**Comment by MaciejFrackowiak** [comment]
If you use Ship with Allegro you don't have to - a mechanism will add a tracking number to the order automatically.

**Comment by tongalalal** [comment]
@MaciejFrackowiak Then if I test it now, how can I add a package to the order, I don't have the actual package data, what should I do?

**Comment by MaciejFrackowiak** [comment]
You can add tracking numbers using [POST /order/checkout-forms/{id}/shipments](https://developer.allegro.pl/documentation/#operation/createOrderShipmentsUsingPOST).

**Comment by tongalalal** [comment]
@MaciejFrackowiak What if I don't have these box numbers, I'm in beta right now, where do I get them?

**Comment by tongalalal** [comment]
![image](https://github.com/user-attachments/assets/479690dc-e54f-4bf1-81ed-ac21fa1ac21a)
@MaciejFrackowiak What is the tracking number here

**Comment by MaciejFrackowiak** [comment]
It should be a valid tracking number for the selected carrier. We do not create it on request.
